src/10_functions.py

```diff
@@ -10,3 +16,7 @@
 
 # YOUR CODE HERE
 
+if(num % 2 == 0):
+    print("Even!")
+else:
+    print("Odd")
```

Comment:
here is where you can call the function you just created (avoiding unnecessary code duplication)
```
if (is_even(num)){
   ....
}
```